AdminWebsite/AdminWebsite/ClientApp/src/app/suitability/services/questionnaire-api.service.ts

```diff
@@ -3,45 +3,121 @@ import { ParticipantSuitabilityAnswerResponse, SuitabilityAnswerResponse } from
+    [IndividualQuestionKeys.AboutYou,
+      'Is there anything you\'d like the court to take into account when it decides which type of hearing will be suitable?'
+    ],
+    [IndividualQuestionKeys.Interpreter, 'Will you need an interpreter for your hearing?'],
+    [IndividualQuestionKeys.Computer, 'Will you have access to a laptop or desktop computer (not a mobile, not a tablet)?'],
+    [IndividualQuestionKeys.Camera, 'Does your computer have a camera and microphone?'],
+    [IndividualQuestionKeys.Internet, 'At the time of your hearing, will the computer be able to access the internet?'],
+    [IndividualQuestionKeys.Room, 'At the time of your hearing, will you have access to a quiet, private room?'],
+    [IndividualQuestionKeys.Consent, 'Would you be content to take part in your hearing by video?']
+  ]);
+
+  readonly questionsRepresentative = new Map<string, string>([
+    [RepresentativeQuestionKeys.AboutYou, 'Is there anything that could affect your ability to take part in a video hearing?'],
+    [RepresentativeQuestionKeys.Room,
+      'Will you have access to a quiet, private room where you can connect to the internet and where your client can sit with you?'],
+    [RepresentativeQuestionKeys.Computer, 'Will you have access to a laptop or desktop computer (not a mobile, not a tablet)?'],
+    [RepresentativeQuestionKeys.Camera, 'Does your computer have a camera and microphone?'],
+    [RepresentativeQuestionKeys.AboutYourClient,
+      'Is there anything that could affect your client\'s ability to take part in a video hearing?'],
+    [RepresentativeQuestionKeys.ClientAttendance, 'Will your client be attending the hearing?'],
+    [RepresentativeQuestionKeys.HearingSuitability, 'Is there anything about this case you think makes it unsuitable for a video hearing?']
+  ]);
+
+  async getSuitabilityAnswers(cursor: string, limit: number): Promise<SuitabilityAnswersPage> {
+    const response = await this.client.getSuitabilityAnswers(cursor, limit).toPromise();
+    const page = new SuitabilityAnswersPage();
+    page.nextCursor = response.next_cursor;
+    page.questionnaires = response.participant_suitability_answer_response.map(item => this.map(item));
+    return page;
+  }
+
+  private map(response: ParticipantSuitabilityAnswerResponse): ParticipantQuestionnaire {
+    return new ParticipantQuestionnaire({
+      displayName: `${response.first_name} ${response.last_name}`,
+      caseNumber: response.case_number,
+      hearingRole: response.hearing_role,
+      representee: response.representee,
+      participantId: response.participant_id,
+      updatedAt: response.updated_at,
+      answers: this.mapAnswerGroups(response.answers,
+        response.hearing_role && response.hearing_role.toLowerCase()
+          .indexOf('solicitor') > -1),
+    });
+  }
+
+  private mapAnswerGroups(answers: SuitabilityAnswerResponse[], isRepresentative: boolean): SuitabilityAnswerGroup[] {
```

Comment:
instead of passing in a representative, what do you think of having a mapper class to pass in?

```typescript
abstract class QuestionsMapper {
   map(questionId: string): string;
}

class RepresentativeMapper extends QuestionsMapper { ... }
```

This would make adding in translations or new common mapping very easy without having to modify the service.

Then you would have something like:
```typescript
this.mapAnswerGroups(response.answers, isRepresentative(response) ? representativeMapper : individualMapper)
```

or better yet:
```typescript
this.mapAnswersGroup(response.answers, this.getMapper(response))
```